docs/developers/retrievability.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_label: Data retrievability and pruning
```

Comment:
```suggestion
sidebar_label: Data retrievability and pruning
description: Methods for rollup developers to ensure historical data access.
```

Reply:
Where's this description used?

Reply:
in this case, just in SEO ([demo page OG info](https://www.opengraph.xyz/url/https%3A%2F%2Fdocs.celestia.org%2Fnodes%2Fcelestia-node-metrics%2F)). if it were ever included in an auto-generated menu in the docs, it would appear similarly to [the descriptions in this DA node category](https://docs.celestia.org/category/data-availability/)